requirements.txt

```diff
@@ -16,3 +16,4 @@ pytest==2.9.1
 requests==2.9.1
 six==1.10.0
 Werkzeug==0.11.8
+beautifulsoup4
```

Comment:
Yes, bs4 is a required package in wikipedia (`pip show wikipedia`).
But let's say some other module uses bs4 (for which we're dependent on wikipedia to install) and then we decide, for some reason, to remove wikipedia!
So, to avoid breaking things, it's better to be explicit and add `beautifulsoup4==4.4.1` :smile: